lib/bup/drecurse.py

```diff
@@ -2,9 +2,17 @@
 from __future__ import absolute_import
 import stat, os
 
-from bup.helpers import add_error, should_rx_exclude_path, debug1, resolve_parent
+from bup.helpers import add_error, should_rx_exclude_path, debug1, resolve_parent, log
 import bup.xstat as xstat
 
+scandir = None
+try:
+    # scandir allows much faster directory traversals, but is
+    # only included in the stanard library from Python 3.3 upwards.
+    import scandir
+except ImportError:
+    log('Warning: scandir support missing; install python-scandir'
```

Comment:
not really sure it's worth it ... after all, people who have happily been using bup would not really care too much, and there's no way to suppress this other than to actually install it ... that's a bit annoying?

OTOH, I guess there's precedent for it since we do similar things with xattr

However, then again, python 3.5 has os.scandir() with all the performance improvements incorporated, I think, so maybe just use that instead of the extra library?